### DISCUSSION TITLE
[vuetify 3.0. beta1] How can I get naturalWidth after image loaded when use VImg component?

### DISCUSSION THREAD

**Discussion by ilezhizhe** [vuetifyjs/vuetify#15073] [Q&A]
How can I get naturalWidth after image loaded when use VImg component?

**Comment by KaelWD** [comment]
```js
<v-img ref="img" @load="onLoad">

function onLoad () {
  img.value?.naturalWidth
}
```

**Reply by ilezhizhe** [reply]
I encountered a strange problem. Sometimes the naturalWidth value is "undefined",sometime it is correct.

**Reply by funkyvisions** [reply]
Same. If I put a setTimeout I can get the right value.  nextTick works some times, but not all the time.  So definitely a timing issue here.  Ugh.
